src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -1640,10 +1641,10 @@ public async Task CheckHttpConnectionFeatures()
                     await hubConnection.StartAsync().OrTimeout();
 
                     var features = await hubConnection.InvokeAsync<object[]>(nameof(TestHub.GetIHttpConnectionFeatureProperties)).OrTimeout();
```

Comment:
Nit: would the following avoid the casts?

```suggestion
                    var features = await hubConnection.InvokeAsync<JsonElement[]>(nameof(TestHub.GetIHttpConnectionFeatureProperties)).OrTimeout();
```